go/vt/vtadmin/api.go

```diff
@@ -975,6 +976,29 @@ func (api *API) DeleteTablet(ctx context.Context, req *vtadminpb.DeleteTabletReq
 	return &vtadminpb.DeleteTabletResponse{Status: "ok"}, nil
 }
 
+func (api *API) GetTopology(ctx context.Context, req *vtadminpb.GetTopologyRequest) (*vtadminpb.GetTopologyResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "API.GetTopology")
+	defer span.Finish()
+
+	c, err := api.getClusterForRequest(req.ClusterId)
+	if err != nil {
+		return nil, err
+	}
+
+	cluster.AnnotateSpan(c, span)
+
+	if err := c.Vtctld.Dial(ctx); err != nil {
+		return nil, err
+	}
+
+	result, err := c.Vtctld.GetTopology(ctx, &vtctldatapb.GetTopologyRequest{})
```

Comment:
we need an RBAC check as well

Reply:
Ooh, good catch.

Reply:
I'm going to make that a cluster resource, as opposed to adding a topology resource.